### PR TITLE
Added navbar component with design

### PR DESCRIPTION
**Pull request by Jayesh-Patidar**

### CODE REVIEW

**Review by atulbhatt-system32 — COMMENTED** [review]
I doubt adding package-lock to gitignore helps

File: components/Navbar/index.js [hunk 1]
```diff
@@ -1,5 +1,37 @@
 import React from "react";
+import styles from "../../styles/Home.module.css";
+import navbarStyles from "../../styles/Navbar.module.css";
```
Comment (atulbhatt-system32) [inline components/Navbar/index.js:3]: We were going to use styled-components

File: styles/Navbar.module.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
+.navbar {
+    background-color: #FAFAFB;
+    padding: 24px 0;
+}
+
+.navItems {
```
Comment (atulbhatt-system32) [inline styles/Navbar.module.css:6]: I'm not a big fan of camelcase in css. I prefer underscore separator

**Comment by Jayesh-Patidar** [comment]
> I doubt adding package-lock to gitignore helps

It should not be committed because it cause merge conflicts.

**Pull request closed without merging**